### PR TITLE
Added proper temperature scaling of logits

### PR DESCRIPTION
**Pull request by jdchang1**
Temperature scaling is needed for getting reference log probs and policy log probs. Added this functionality to both online and offline pipelines.

### CODE REVIEW

**Review by gupta-abhay — APPROVED** [review]
lgtm!!
